dcc-storage-client/src/main/java/org/icgc/dcc/storage/client/download/DownloadStateStore.java

```diff
@@ -110,9 +129,9 @@ private Part loadPart(File stateDir, String objectId, String partFileName) {
     File objectStateDir = getObjectStateDir(stateDir, objectId);
     File partFile = new File(objectStateDir, partFileName);
 
-    ObjectMapper mapper = new ObjectMapper();
+    val mapper = new ObjectMapper();
     Part part = null;
-    try (FileInputStream partStream = new FileInputStream(partFile)) {
+    try (val partStream = new FileInputStream(partFile)) {
       part = mapper.readValue(partStream, Part.class);
```

Comment:
Can just use `mapper.readValue(partFile, Part.class)`. Would encapsulate this in a helper (i.e. `readPart(partFile))`